<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlBar</class>
 <widget class="QWidget" name="CtrlBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>80</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1677721</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="fast_backward_btn">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VideoSlider" name="progress_slid">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fast_forward_btn">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speed_lb">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>倍速:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="speed_cb">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vol_lb">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>音量：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VideoSlider" name="vol_slid">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="play_btn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pre_btn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>上一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>下一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_btn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="now_time_lb">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="split_lb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="total_time_lb">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>99:99:99</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoSlider</class>
   <extends>QSlider</extends>
   <header location="global">videoslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
